pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -919,8 +907,7 @@ pyramid_oereb:
       #        - SubTheme2
```

Comment:
we could remove this too, as there ar no subthemes in this topic or move the comments to the LandUsePlan where SubThemes are allowed